tox.ini

```diff
@@ -4,15 +4,11 @@
 minversion = 3.18
```

Comment:
I think we should update this to `minversion = 4.0` in `meta/config`, so developers do not run into problem when using `tox < 4` locally.

Reply:
Created PR https://github.com/zopefoundation/meta/pull/188